src/layout/BottomBarItemInsert.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "@emotion/styled";
+import theme from "../theme";
+
+type BottomBarItemInsertProps = {
+  onClick?: (e: React.SyntheticEvent<Element, Event>) => any | void;
+  grab?: boolean;
+};
+
+export const BottomBarItemInsert = styled.button<BottomBarItemInsertProps>`
+  border: none;
+  padding: none;
+  background: transparent;
+  align-content: center;
+  color: ${theme.colors.darkGrey};
+  &:hover {
+    color: ${theme.colors.heading};
+    cursor: ${p => (p.grab ? "grab" : "pointer")};
+  }
+  &:active {
+    cursor: ${p => (p.grab ? "grabbing" : "pointer")};
+  }
+`;
```

Comment:
Can we add new line at eof?

Reply:
Changed in [this commit](https://github.com/onflow/flow-playground/pull/175/commits/289b5d04ba186213347bb27b87021fdfc8efaf3e)